crates/sp-domains-fraud-proof/src/storage_proof.rs

```diff
@@ -414,16 +414,27 @@ impl MaybeDomainRuntimeUpgradedProof {
 }
 
 #[derive(Clone, Debug, Decode, Encode, Eq, PartialEq, TypeInfo)]
-pub struct DomainInherentExtrinsicDataProof {
+pub struct InvalidExtrinsicsDataProof {
```

Comment:
Non-blocker but it may be worth having `Inherent` in the name, to distinguish it from the invalid extrinsic from the invalid bundle fraud proof.